Dockerfile

```diff
@@ -0,0 +1,9 @@
+FROM python:3.6-alpine
+
+WORKDIR "/root"
+COPY bowser /root/bowser
+COPY scripts /root/scripts
+COPY setup.py /root/setup.py
+RUN pip install -e .
```

Comment:
For testing installing from the src dir might not be as ideal, you'd want to quickly iterate over many small versions so installing during the image build could be problematic.